dotcom-rendering/src/client/bootCmp.ts

```diff
@@ -81,6 +81,47 @@ const submitConsentEventsToOphan = (renderingTarget: RenderingTarget) =>
 		return submitComponentEvent(event, renderingTarget);
 	});
 
+const submitConsentToOphan = (renderingTarget: RenderingTarget) =>
+	onConsent().then((consentState: ConsentState) => {
+		// eslint-disable-next-line @typescript-eslint/strict-boolean-expressions, @typescript-eslint/no-unnecessary-condition -- Review if this check is needed
+		if (!consentState) return;
+		// Register changes in consent state with Ophan
+
+		let consentJurisdiction = 'Other';
+		let consentUUID = '';
+		let consent = '';
+
+		if (consentState.tcfv2) {
+			consentJurisdiction = 'TCF';
+			consentUUID = getCookie({ name: 'consentUUID' }) ?? '';
+			consent = consentState.tcfv2.tcString;
+		}
+		if (consentState.ccpa) {
+			consentJurisdiction = 'CCPA';
+			consentUUID = getCookie({ name: 'ccpaUUID' }) ?? '';
+			consent = consentState.ccpa.doNotSell ? 'true' : 'false';
+		}
+		if (consentState.aus) {
+			consentJurisdiction = 'AUS';
+			consentUUID = getCookie({ name: 'ccpaUUID' }) ?? '';
+			/*consent =
+					getCookie({ name: 'consentStatus' }) ?? '';
+					*/
+			consent = consentState.aus.personalisedAdvertising
+				? 'true'
+				: 'false';
+		}
```

Comment:
While this is not a hard and fast rule, I think that avoiding mutation here by returning early in each of the `if` blocks would simplify the logic a lot. I also think this processing function could be extracted:

```ts

const formatConsent = (consentState: ConsentState): { jurisdiction: 'TCF' | 'CCPA' | 'AUS' , uuid: string, consent: boolean | string } => {
  if(consent.tcfv2) {
    return {
      jurisdiction: 'TCF',
      uuid: getCookie({ name: 'consentUUID' }) ?? '',
      consent: consentState.tcfv2.tcString,
    }
  }
  if(consent.ccpa) {
    return {
      jurisdiction: 'CCPA',
      uuid: getCookie({ name: 'ccpaUUID' }) ?? '',
      consent: !consentState.ccpa.doNotSell,
    }
  }
  // etc.
}

```